tests/test_models.py

```diff
@@ -3122,12 +3122,19 @@ def test_no_parent(self):
     def test_get_recent_logs(self):
         # Add some logs
         for _ in range(5):
-            self.project.logs.append(NodeLogFactory())
+            NodeLogFactory.create(
```

Comment:
This is a lot of kwargs to pass in every time. It might be better to define a `_create` for [NodeLogFactories](https://github.com/CenterForOpenScience/osf.io/blob/develop/tests/factories.py#L300). Something like

```python
@classmethod
def _create(cls, *args, **kwargs):
    node = kwargs.get('node') or NodeFactory()
    user = kwargs.get('user') or node.creator
    original_node = kwargs.get('original_node') or node
    # etc.
    return NodeLog(**locals)
```